app/views/blogs/new.html.haml

```diff
@@ -0,0 +1,25 @@
+- if !user_signed_in?
```

Comment:
No need to check if user is signed in since it is already checked in the controller.